index.html

```diff
@@ -155,7 +204,8 @@ <h1>Multi-Post
                </ul>
                <div class="external_link">
                    <a href="#">See Project</a>
-               </div>
+               
+              </div>
```

Comment:
- [ ]  Kyndly stick to the figma design as much as posible please notice you have several diferences in the About myself section .

## Your design 
![image](https://user-images.githubusercontent.com/95243996/201199537-43017528-be50-4938-968a-00cd415e403f.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95243996/201199418-a5acc49b-b57e-4043-b3c1-e2b205381835.png)